sdk/include/psdk/ntgdi.h

```diff
@@ -1598,12 +1598,12 @@ W32KAPI
 BOOL
 APIENTRY
 NtGdiRemoveFontResourceW(
-    _In_reads_(cwc) WCHAR *pwszFiles,
+    _In_reads_(cwc) const WCHAR *pwszFiles,
```

Comment:
```suggestion
    _In_reads_(cwc) PCWSTR pwszFiles,
```
or PCWCH if it's a non-NUL-terminated string.

Reply:
```suggestion
    _In_reads_(cwc) PCWCH pwszFiles,
```